br/tests/br_file_corruption/run.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
```

Comment:
can we also add scenarios where sst file or log file is missing and is out of sync with backup meta file?